tests/integration/test_format_txts_e2e.py

```
@@ -0,0 +1,266 @@
+"""End-to-end integration tests for format_txts.py."""
+
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)